KEEP/initramfs.sh

```diff
@@ -66,4 +67,3 @@ umount /sys /proc
 [ -d /boot ] && umount /boot
 mount --move /dev /root/dev
 exec switch_root /root "${init}"
-rescue "switch_root failed with $?"
```

Comment:
exec XYZ will only return if the startup/loading of XYZ fails, if the program gets executed (which is practically always the case), it will never return, even if the program itself fails badly.